search/processor.cpp

```diff
@@ -619,6 +620,26 @@ bool Processor::SearchCoordinates()
     }
   }
 
+  {
+    double lat;
+    double lon;
```

Comment:
I'd prefer to declare double lat, lon; once on top. And remove useless block braces {}

Reply:
Removed extra `{}` and don't declare `double lat, lon` multiple times.